eng/scripts/Create-APIView.ps1

```diff
@@ -49,13 +96,32 @@ function Get-SwaggerReadMeFile {
 #>
 function Get-ResourceProviderFromReadMePath {
```

Comment:
What do you end up using the resource provider name for?

Reply:
Can we just use the path to the readme as any key?

Reply:
It becomes the name of the APIView